darts/models/forecasting/baselines.py

```diff
@@ -177,18 +178,26 @@ def __init__(
 
         Parameters
         ----------
-        models
+        forecasting_models
             List of forecasting models whose predictions to ensemble
+        retrain_forecasting_models
```

Comment:
Can you add here that NaiveEnsemble doesn't have to be trained with retrain=False and predict can be call directly?